src/main/java/nextstep/subway/linebridge/domain/LineBridges.java

```diff
@@ -0,0 +1,107 @@
+package nextstep.subway.linebridge.domain;
+
+import nextstep.subway.station.domain.Station;
+
+import java.util.LinkedList;
+import javax.persistence.CascadeType;
+import javax.persistence.Embeddable;
+import javax.persistence.OneToMany;
+import java.util.List;
+import java.util.Optional;
+
+@Embeddable
+public class LineBridges {
+
+    public static final int ONLY_HAS_ONE_ENTRY = 1;
+
+    @OneToMany(mappedBy = "line", cascade = CascadeType.ALL)
+    private List<LineBridge> lineBridges = new LinkedList<>();
+
+    public void add(LineBridge lineBridge) {
+        validateDuplicate(lineBridge);
+        validateNotingMatch(lineBridge);
+
+        repairSections(lineBridge);
+        lineBridges.add(lineBridge);
+    }
+
+    private void validateDuplicate(LineBridge lineBridge) {
+        Optional<LineBridge> duplicate = lineBridges.stream()
+                .filter(element -> element.isSame(lineBridge))
+                .findFirst();
+
+        if (duplicate.isPresent()) {
+            throw new IllegalArgumentException("상행역과 하행역이 이미 노선에 등록되어 있으면 구간을 추가할 수 없습니다.");
+        }
+    }
```

Comment:
`stream`을 이용하는 만큼 `stream`의 장점을 활용하면 좋을듯해요!

[Java Optional 바르게 쓰기](https://homoefficio.github.io/2019/10/03/Java-Optional-%EB%B0%94%EB%A5%B4%EA%B2%8C-%EC%93%B0%EA%B8%B0/)


```suggestion
    private void validateDuplicate(LineBridge lineBridge) {
             lineBridges.stream()
              .filter(element -> element.isSame(lineBridge))
              .findFirst().orElseThrow(() -> new IllegalArgumentException("상행역과 하행역이 이미 노선에 등록되어 있으면 구간을 추가할 수 없습니다."));
        }
    }
```